fud/bitstream/sim_script.tcl

```diff
@@ -0,0 +1,13 @@
+add_wave {{/emu_wrapper/emu_i/Toplevel_1/inst/*}}
```

Comment:
Is this file generic?

Reply:
It's generic except for the memory interfaces (it only adds one for now)